nightly_build.sh

```diff
@@ -81,8 +81,11 @@ source env.sh
 cd $INGEST_PATH
 export CLINICAL_DATA_LOCATION=$INGEST_PATH/tests/clinical_ingest.json
 # should be pip install -r requirements.txt, but that didn't seem to work last I checked -- dependency errors?
+# `tox 4.12.1 requires colorama>=0.4.6, but you have colorama 0.4.4 which is incompatible.`
+git pull
 pip install dateparser
 pip install openapi_spec_validator
+pip install clinical_etl@git+https://github.com/CanDIG/clinical_ETL_code.git@v2.0.0
```

Comment:
Is there a reason you're not doing `pip install -r etc/venv/requirements.txt` here?

Reply:
Trying to run that crashes with `tox 4.12.1 requires colorama>=0.4.6, but you have colorama 0.4.4 which is incompatible.`. There might be a better fix somehow, but I'm unsure how to resolve

Reply:
weird...if you look at pip list now, it seems like it has colorama 0.4.6 installed.